core/src/db/entity/metadata/media_metadata.rs

```diff
@@ -151,6 +156,12 @@ pub struct MediaMetadata {
 	/// the number of pages detected by the file processor.
 	#[serde(alias = "PageCount")]
 	pub page_count: Option<i32>,
+	/// The (height, width) dimensions of each page in the associated media. This is
+	/// generated by the image analysis job and is [None] before being generated.
+	#[serde(alias = "PageDimensions", default = "Option::default")]
```

Comment:
`PageDimensions` isn't actually part of the [ComicInfo.xml schema](https://github.com/anansi-project/comicinfo/blob/main/schema/v2.0/ComicInfo.xsd), so this alias won't be used for the time being. I think it Stump ever supports writing metadata, it could be useful though! It would _technically_ deviate from the schema, but I think this would be acceptable.

Also, there _is_ a [`Pages` field](https://github.com/anansi-project/comicinfo/blob/main/schema/v2.0/ComicInfo.xsd#L98C41-L98C50) in that schema. It has an attribute called `ImageSize`, which I think is less useful than dimensions.

None of this is to say I'm asking for changes, to be clear, just vomiting some context 😅

Reply:
That's a good note, I realized halfway through this process that `MediaMetatdata` was doing double duty as a database object representation and also a deserialization type for xml metadata. I didn't appreciate the reasoning for "alias" so I just mirrored it.

I think it might be a good idea to make two types in the future - one that represents the DB object and another that explicitly represents ComicInfo.xml.

In the meantime, I'll remove the unnecessary alias.

Reply:
I also added a `skip_deserializing` since it shouldn't ever be contained in the xml.